src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMath.cs

```diff
@@ -110,23 +140,30 @@ public override FormulaValue GetResult(IRContext irContext)
         }
             
         private class MinNumberAgg : IAggregator
-        {
+        {            
             protected double _minValue = double.MaxValue;
+            protected static readonly double _defaultN = 0d;
 
             public void Apply(FormulaValue value)
-            {
-                if (value is BlankValue)
-                {
-                    return;
-                }
+            {             
+                var n1 = value is BlankValue ? _defaultN : ((NumberValue)value).Value;
 
-                var n1 = (NumberValue)value;
-                if (n1.Value < _minValue)
+                if (n1 < _minValue)
                 {
-                    _minValue = n1.Value;
+                    _minValue = n1;
                 }
             }
 
+            public FormulaValue NoElementValue(IRContext context)
+            {
+                return GetDefault(context);
+            }
+
+            public FormulaValue GetDefault(IRContext context)
+            {
+                return new NumberValue(context, 0);
```

Comment:
>return new NumberValue(context, 0) [](http://example.com/codeflow?start=16&length=34)

Similar comment than the one for Sum; should we return 0? 0 isn't really a "neutral" value for min.